hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -950,6 +950,10 @@
 
   public static final int REPLICATION_SOURCE_TOTAL_BUFFER_DFAULT = 256 * 1024 * 1024;
 
+  /** Configuration key for ReplicationSource shipeEdits timeout */
+  public static final String REPLICATION_SOURCE_SHIPEDITS_TIMEOUT =
+      "replication.source.shipedits.timeout";
```

Comment:
Usually the configuration key is started with "hbase."

Reply:
Since most of the conf keys for ReplicationSource and ReplicationSink are prefixed with replicaiton, I think this is fine.

Reply:
OK, keep the existing naming style firstly